src/components/ui/Chat/MessageAttachment/Styled.tsx

```diff
@@ -0,0 +1,53 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import styled from 'styled-components';
+import { baseStyles, baseSpacing } from '../../Base';
+
+export const StyledMessageAttachmentContent = styled.div`
+  color: ${(props) => props.theme.colors.greys.grey80};
```

Comment:
We will want to put color properties into the light and dark theme files, even if we don't know the dark theme value.

Reply:
oops missed it! Thanks